helper-cli/build.gradle.kts

```diff
@@ -43,4 +44,8 @@ dependencies {
     implementation(libs.jslt)
     implementation(libs.log4jApi)
     implementation(libs.slf4j)
+
+    "pluginClasspath"(platform(project(":plugins:package-configuration-providers")))
+    "pluginClasspath"(platform(project(":plugins:package-curation-providers")))
+    "pluginClasspath"(platform(project(":plugins:version-control-systems")))
```

Comment:
@fviernau if there are more [plugin types](https://github.com/oss-review-toolkit/ort/tree/main/plugins) the helper CLI requires at runtime, they also need to be added here.

Reply:
I'll need to investigate this a bit. Shall we merge this one already before?

Reply:
I first need to fixup an compile issue. I'd say then test it, and merge yourself if your find it to resolve your issue.

Reply:
Thank you! I've just tested it, and verified it successfully downloads via `git` now.